d365fo.tools/functions/get-d365labelidvalid.ps1

```diff
@@ -0,0 +1,49 @@
+<#
+    .SYNOPSIS
+        Checks if a string is a valid 'Label Id' format
+ 
+    .DESCRIPTION
+        This function will validate if a string is a valid 'Label Id' format.
+
+    .PARAMETER LabelId
+        LabelId string to validate
+ 
+    .EXAMPLE
+        Get-D365LabelIdIsValid -LabelId "ABC123" #True
```

Comment:
We always prefix an example like this:
PS C:\> Get-D365LabelIdIsValid...

This is to make it feel more like the real thing when beginner are trying to learn how to use the module.